spec/setup_acceptance_node.pp

```diff
@@ -45,13 +45,20 @@
 $client_cert = "${directory}/client-cert.pem"
 $client_key = "${directory}/client-key.pem"
 
+# EL7 lacks openssl -addext
+if $facts['os']['release']['major'] == '7' {
+  $ca_cmd = "openssl req -nodes -x509 -newkey rsa:2048 -subj '/CN=${facts['networking']['fqdn']}' -keyout '${ca_key}' -out '${ca_cert}' -days 365"
```

Comment:
You could opt to deploy a file with extensions defined in it similar to https://github.com/theforeman/foreman-installer/blob/develop/spec/fixtures/katello-certs-check/extensions.txt

And then point at the extensions when running this command similar to https://github.com/theforeman/foreman-installer/blob/develop/spec/fixtures/katello-certs-check/create_cert.sh#L36

Reply:
I considered that, but how long do we plan on supporting EL7? It is mostly needed in the pulp-cli work from #252 but IMHO there's it's sufficient to only check stderr on EL8.

Reply:
#252 passes with this change picked on EL8, so I'm happy enough.

(On the other hand, we could consider limiting the number of ways we generate certs in our CI…)